setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup, find_packages
+
+NAME = 'aftquery'
+DESCRIPTION = 'A command line tools for discovering info about tennis tournaments and players in Belgiium'
```

Comment:
I don't write lines longer than 80 characters. Please fix it. In general, I'm following quite strictly pep-8 and also the automatic formatting of the **black** tool.

Reply:
```suggestion
DESCRIPTION = 'A command line tools for discovering info about '
              'tennis tournaments and players in Belgium'
```